layouts/partials/site-header.html

```diff
@@ -66,7 +66,12 @@ <h3 class="font-bold text-3xl lg:text-5xl">{{ .Title }}</h3>
                 </div>
                 {{ if .Params.subheadertext }}
                 {{ with .Params.subheadertext }}
-                <p class="text-2xl lg:text-4xl mt-1 lg:mt-2 font-extralight">{{ . }}</p>
+                <p class="hidden sm:block text-2xl lg:text-4xl mt-1 lg:mt-2 font-extralight">{{ . }}</p>
+                {{ end }}
+                {{ end }}
+                {{ if .Params.moresubheadertext }}
+                {{ with .Params.moresubheadertext }}
```

Comment:
`moresubheadertext` was added to provide a little more control over how the wording appears.